src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_webapp_commands.py

```diff
@@ -136,10 +136,10 @@ def test_win_webapp_quick_create(self, resource_group):
         r = self.cmd('webapp create -g {} -n {} --plan {} --deployment-local-git'.format(
             resource_group, webapp_name, plan)).get_output_in_json()
         self.assertTrue(r['ftpPublishingUrl'].startswith('ftp://'))
-        self.cmd('webapp config appsettings list -g {} -n {}'.format(resource_group, webapp_name, checks=[
+        self.cmd('webapp config appsettings list -g {} -n {}'.format(resource_group, webapp_name), checks=[
             JMESPathCheck('[0].name', 'WEBSITE_NODE_DEFAULT_VERSION'),
-            JMESPathCheck('[0].value', '8.11.1'),
-        ]))
+            JMESPathCheck('[0].value', '10.14'),
+        ])
```

Comment:
good catch! Cannot believe the original test can pass the CI

Reply:
Well, `str.format` will ignore the extra positional pramaeter values, so it never checks.